Demo_Project/src/pages/Login.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from "react";
+import { Link, Redirect } from "react-router-dom";
+import { makeStyles, Typography } from "@material-ui/core";
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+import ErrorOutlineIcon from "@material-ui/icons/ErrorOutline";
+import Tooltip from "@material-ui/core/Tooltip";
+import Popover from "@material-ui/core/Popover";
+import axios from "axios";
+import USER from '../GlobalData/user';
+const useStyles = makeStyles(theme => ({
+  root: {
+    flexGrow: 1,
+    textAlign: "center"
+  },
+  appBar: {
+    backgroundColor: "#ffffff",
+    maxWidth: "100^%",
+    boxShadow: "none"
+  },
+  menuButton: {
+    marginRight: theme.spacing(2)
+  },
+  abeld: {
+    color: "0d0d0d"
+  },
+  disabeld: {
+    color: "#b2aaaa"
+  },
+  title: {
+    flexGrow: 1,
+    fontFamily: "Titillium Web",
+    fontSize: 50
+  },
+  subTitle: {
+    color: "#5CB85C"
+  },
+  textField: {
+    width: "400px"
+  },
+  submitButton: {
+    backgroundColor: "#5CB85C",
+    color: "white",
+    width: "100px",
+    height: "50px"
+  },
+  error: {
+    color: "red"
+  },
+  typography: {
+    padding: theme.spacing(2)
+  }
+}));
+function Login() {
+  const userobj= new USER()
+  const classes = useStyles();
+  const [user,setUser]=useState({})
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [errorMas, setErrorMas] = useState("");
+  const open = Boolean(anchorEl);
+  const id = open ? "simple-popover" : undefined;
+  const handelLogin = (event: any) => {
```

Comment:
now you only have one layer in your app structure (which is UI),
create a new layer for network/api calls to handle stuff like this, this shouldn't be handled by the UI function, it should be separate